e2e/tests/truffle/test/helpers.js

```diff
@@ -18,6 +18,12 @@ async function delay(delayInms) {
   });
 }
 
+function toHexString(byteArray) {
```

Comment:
Does `Buffer.from(byteArray).toString('hex')` not work?

Reply:
yes it works, fixed